jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/RoadSideUnit.java

```diff
@@ -110,6 +115,27 @@ public void setSnmpProtocol(SnmpProtocol snmpProtocol) {
             this.snmpProtocol = snmpProtocol;
         }
 
+        public void setDefaultSnmpProtocol() {
+            String defaultSnmpProtocol = System.getenv("DEFAULT_SNMP_PROTOCOL");
+            if (defaultSnmpProtocol != null) {
+                switch (defaultSnmpProtocol) {
+                    case "FOURDOT1":
+                        this.snmpProtocol = SnmpProtocol.FOURDOT1;
+                        break;
+                    case "NTCIP1218":
+                        this.snmpProtocol = SnmpProtocol.NTCIP1218;
+                        break;
+                    default:
+                        logger.error("Unrecognized SNMP Protocol: {}, defaulting to FOURDOT1", defaultSnmpProtocol);
+                        this.snmpProtocol = SnmpProtocol.FOURDOT1;
+                        break;
+                }
+            } else {
+                logger.info("No SNMP Protocol specified, using FOURDOT1");

```

Comment:
As discussed in our closeout meeting, let's pop this default over to the 1218 instead of 4.1.

Reply:
Swapped this to 1218